v2/support/garden_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,105 @@
+<html>
+<head>
+  <link rel="stylesheet" href="https://assets.zendesk.com/apps/sdk-assets/css/0/zendesk_garden.css" type="text/css">
```

Comment:
Since we're currently at 0.2.1, so specifying just "0" will return the latest 0.x.x version.